### DISCUSSION TITLE
How you pass proxy env variables to slave containers?

### DISCUSSION THREAD

**Discussion by JannikRosendahl** [nextcloud/all-in-one#3876] [Questions] [answered]
Hello,
I am running Nextcloud AIO in a DMZ where machines need to use a proxy to access the web. I have configured the docker daemon on my host VM accordingly in `~/.docker/config.json` and I am also passing the environment variables to the master-container via my compose file. I can confirm that the variables are set correctly by inspecting the container. 
However, the _nextcloud-aio-nextcloud_ container does not seem to use these variables, because of this, the container takes a long time to start up as it seems to be running into a timeout. This is the beginning of the log of the startup of the _nextcloud-aio-nextcloud_ container.
```
Installing imagemagick via apk...
WARNING: fetching https://dl-cdn.alpinelinux.org/alpine/v3.18/main: Permission denied
WARNING: fetching https://dl-cdn.alpinelinux.org/alpine/v3.18/community: Permission denied
ERROR: unable to select packages:
  imagemagick (no such package):
    required by: world[imagemagick]
The packet imagemagick was not installed!
Enabling Imagick...
WARNING: fetching https://dl-cdn.alpinelinux.org/alpine/v3.18/main: Permission denied
WARNING: fetching https://dl-cdn.alpinelinux.org/alpine/v3.18/community: Permission denied
ERROR: unable to select packages:
  .docker-php-ext-enable-deps-20231205.173636:
    masked in: cache
    satisfies: world[.docker-php-ext-enable-deps=20231205.173636]
  binutils (no such package):
    required by: .docker-php-ext-enable-deps-20231205.173636[binutils]
Could not install PHP extension imagick!
Configuring Redis as session handler...
...
```
Can I somehow pass these proxy env variables to the containers managed by the mastercontainer?

**Comment by szaimen** [comment, marked as answer]
Was answered in https://help.nextcloud.com/t/aio-with-environmental-variables-for-derived-containers/175836/2
